src/Microsoft.IdentityModel.Clients.ActiveDirectory/Platforms/android/AndroidBrokerProxy.cs

```diff
@@ -286,12 +302,12 @@ private Account FindAccount(string accountName, Account[] accountList)
             {
                 foreach (Account account in accountList)
                 {
-                    bool found = account != null && 
-                                 account.Name != null && 
+                    bool found = account != null &&
+                                 !string.IsNullOrEmpty(account.Name) &&
```

Comment:
>!string.IsNullOrEmpty(account.Name) && [](start = 33, length = 38)

what about testing for accountName == null? (perhaps as input param validation)

Reply:
in the two places this is used, we check for a null accountName before going into FindAccount()

---
In reply to: [284967266](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1598#discussion_r284967266) [](ancestors = 284967266)